CMakeLists.txt

```diff
@@ -50,10 +49,23 @@ if(NOT ${use_installed_dependencies})
     if ((NOT TARGET aziotsharedutil) AND (EXISTS ${CMAKE_CURRENT_LIST_DIR}/deps/azure-c-shared-utility/CMakeLists.txt))
         add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/deps/azure-c-shared-utility)
     endif()
+else()
+    if (NOT azure_macro_utils_cFOUND)
+        find_package(azure_macro_utils_c REQUIRED CONFIG)
+    endif ()
+    if (NOT umock_cFOUND)
+        find_package(umock_c REQUIRED CONFIG)
+    endif ()
+    if (NOT azure_c_shared_utility_FOUND)
```

Comment:
why is this different from the rest of the variables of this format?

Reply:
you mean the trailing _ should be removed. Thanks, that's correct